packages/channel-utils/src/insights-utils.ts

```diff
@@ -36,19 +36,35 @@ export const saveAccounts = async (
         update: {
           currency: acc.currency,
           name: acc.name,
-          integrations: { connect: { id: integration.id } },
           organizations: { connect: { id: integration.organizationId } },
         },
         create: {
-          integrations: { connect: { id: integration.id } },
           externalId: acc.externalId,
           currency: acc.currency,
           name: acc.name,
           type: integration.type,
           organizations: { connect: { id: integration.organizationId } },
```

Comment:
```suggestion
          organizations: { connect: { id: integration.organizationId } },
                    adAccountIntegrations: {
            create: {
              integrationId: integration.id,
              enabled: true,
            },
          }
```
for your consideration...